packages/generators/addon-generator.ts

```diff
@@ -39,12 +39,9 @@ export default function addonGenerator(
 		public copy: (value: string, index: number, array: string[]) => void;
 		public copyTpl: (value: string, index: number, array: string[]) => void;
 
-		public prompting(): Promise<{}> {
-			return this.prompt(prompts).then(
-				(props: InquirerScaffoldObject): void => {
-					this.props = props;
-				}
-			);
+		public async prompting() {
```

Comment:
I don't know if you have rebased from master, but now with eslint we force defining a return type. So please use `Promise<void>`

Reply:
😮 something definitely is wrong with my editor.

Fixed this, thanks for your patience on this @ematipico ✨